pages.fr/common/pip3.md

```diff
@@ -0,0 +1,36 @@
+# pip3
+
+> Gestionnaire des paquets pour Python.
+> Plus d'informations: <https://pip.pypa.io>.
+
+- Recherche un paquet :
+
+`pip3 search {{paquet}}`
+
+- Installe un paquet :
+
+`pip3 install {{paquet}}`
+
+- Installe une version particulière d'un paquet :
+
+`pip3 install {{paquet}}=={{version}}`
+
+- Met à jour un paquet :
+
+`pip3 install --upgrade {{paquet}}`
+
+- Désinstalle un paquet :
+
+`pip3 uninstall {{paquet}}`
+
+- Sauvegarde une liste des paquets installés :
+
+`pip3 freeze > {{requirements.txt}}`
+
+- Installe des paquets à partir d'un fichier :
+
+`pip3 install --requirements {{requirements.txt}}`
+
+- Affiche les informations d'un paquet installé :
+
+`pip3 show {{package_name}}`
```

Comment:
I think we can't translate `requirements.txt` in the previous example because in like the `package.json` file for npm. But `package_name` could be translated, right?

Reply:
Yes!  Missed that one, thanks.